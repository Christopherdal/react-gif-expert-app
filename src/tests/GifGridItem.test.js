import React from 'react';
// import { render } from '@testing-library/react';
import { shallow } from 'enzyme';
import { GifGridItem } from './../shared/components/gif/GifGridItem.component';

describe('Test on GiftGridItem Component', () => {
	let wrapper = shallow(<GifGridItem />);

	test('should Match With Snapshot GiftGridItem Correctly', () => {
		expect(wrapper).toMatchSnapshot(); // To Save the literal rendered component and compare with wrapper if you press 'u' update the snapshot
	});
});
