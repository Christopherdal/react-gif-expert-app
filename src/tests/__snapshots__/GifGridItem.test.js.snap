// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on GiftGridItem Component should Match With Snapshot GiftGridItem Correctly 1`] = `
<div
  className="card animate__animated animate__bounce"
>
  <img />
  <p>
     
     
  </p>
</div>
`;
